verilog/tools/ls/README.md

```diff
@@ -236,3 +271,55 @@ code --install-extension verible.vsix
 ```
 
 [release]: https://github.com/chipsalliance/verible/releases
+
+## Configuring the Language Server for a project
+
+### Adding `verible.filelist` for project-wide symbol discovery
+
+`verible-verilog-ls` by default loads and analyses only currently edited files in the editor.
+To be able to utilize such features as going to definition or references project-wide, add a `verible.filelist` file to the project.
+`verible.filelist` is a file containing a list of design files to collect data necessary to build a symbol table and other helper structures to allow Language Server to provide definitions, references, information on hover and more.
+It is a simple file containing one file per line - it can be easily created using such tools as `find`, e.g.:
+
+```bash
+find . -name "*.sv" -o -name "*.svh" -o -name "*.v" | sort > verible.filelist
+```
+
+With it, the Language Server creates a symbol table for the entire project, allowing to find definitions and references for a symbol accross multiple files.
```

Comment:
allowing to find definitions is now mentioned the third time in this paragraph, so this feels redundant.